fastai/vision/gan.py

```diff
@@ -298,8 +364,21 @@ def _tk_diff(real_pred, fake_pred): return real_pred.mean() - fake_pred.mean()
 @delegates()
 class GANLearner(Learner):
     "A `Learner` suitable for GANs."
-    def __init__(self, dls, generator, critic, gen_loss_func, crit_loss_func, switcher=None, gen_first=False,
-                 switch_eval=True, show_img=True, clip=None, cbs=None, metrics=None, **kwargs):
+    def __init__(self,
+        dls:DataLoaders, # DataLoaders object for GAN data
+        generator:nn.Module, # Generator model
+        critic:nn.Module, # Critic model
+        gen_loss_func:callable, # Generator loss function
+        crit_loss_func:callable, # Critic loss function
+        switcher:(Callback,None)=None, # Callback for switching between generator and critic training, defaults to `FixedGANSwitcher`
+        gen_first:bool=False, # Whether we start with generator training
+        switch_eval:bool=True, # Whether the model should be set to eval mode when calculating loss
+        show_img:bool=True, # Whether to show example generated images during training
+        clip:(None, float)=None, # How much to clip the weights
+        cbs:(Callback, None, list)=None, # Additional callbacks
+        metrics:(None, list, callable)=None, # Metrics
```

Comment:
```suggestion
        metrics:(None, list, callable)=None, # `Metrics`
```
Hoping this will link to the fastai metrics doc, otherwise remove this comment

Reply:
It apparently doesn't